warp10/src/main/java/io/warp10/script/WarpScriptLib.java

```diff
@@ -1461,9 +1481,20 @@
 
     // Crypto functions
     addNamedWarpScriptFunction(new HASH(HASH));
+    addNamedWarpScriptFunction(new DIGEST(MD2, MD2Digest.class));
+    addNamedWarpScriptFunction(new DIGEST(MD4, MD4Digest.class));
     addNamedWarpScriptFunction(new DIGEST(MD5, MD5Digest.class));
     addNamedWarpScriptFunction(new DIGEST(SHA1, SHA1Digest.class));
     addNamedWarpScriptFunction(new DIGEST(SHA256, SHA256Digest.class));
+    addNamedWarpScriptFunction(new DIGEST(SHA224, SHA224Digest.class));
+    addNamedWarpScriptFunction(new DIGEST(SHA384, SHA384Digest.class));
+    addNamedWarpScriptFunction(new DIGEST(SHA512, SHA512Digest.class));
+    addNamedWarpScriptFunction(new DIGEST(RIPEMD128, RIPEMD128Digest.class));
+    addNamedWarpScriptFunction(new DIGEST(RIPEMD160, RIPEMD160Digest.class));
+    addNamedWarpScriptFunction(new DIGEST(RIPEMD256, RIPEMD256Digest.class));
+    addNamedWarpScriptFunction(new DIGEST(RIPEMD320, RIPEMD320Digest.class));
+    addNamedWarpScriptFunction(new DIGEST(GOST, GOST3411Digest.class));
```

Comment:
Seems there are also Tiger and Whirlpool digests available. I don't know it there are worth to to be added, though.